k-distribution/tests/regression-new/checks/signature.k.out

```diff
@@ -0,0 +1,28 @@
+[Error] Inner Parser: Parse error: unexpected token 'foo'.
+	Source(signature.k)
+	Location(24,8,24,11)
+[Error] Inner Parser: Parse error: unexpected token 'fu'.
+	Source(signature.k)
+	Location(26,8,26,10)
+[Error] Inner Parser: Parse error: unexpected token 'baz'.
+	Source(signature.k)
+	Location(28,8,28,11)
```

Comment:
Can we make these error messages more informative easily? Something like also saying "Symbol 'baz' is part of the private signature of module 'A', and so cannot be used directly in module 'B'"?

Reply:
Not easily. We would have to have some way of recognizing that it parsed as a "baz" and then rejecting that parse, which is going to be a lot less efficient than simply removing it from the grammar entirely.